Update Introduction_1_0.md (#215)
whamcloud/Online-Help: docs/Introduction_1_0.md

- [Key Features](#key-features)
- [Management mode versus Monitor-only mode](#management-mode-versus-monitor-only-mode)
- [Overview of the graphical user interface](#overview-of-the-graphical-user-interface)

>   <!-- - [Menu bar](#menu-bar)-->

Reviewer: If this is commented out, I think it's worth removal.

Reviewer 2: All comments have been removed.

## Overview of Integrated Manager for Lustre software

> Integrated Manager for Lustre software is a global single-namespace file system architecture that allows parallel access by many clients to all the data in the file system across many servers and storage devices. Designed to take advantage of the reliability features of enterprise-class storage hardware, Integrated Manager for Lustre software supports availability features such as redundant servers with storage failover. Metadata and data are stored on separate servers to allow each system to be optimized for the different workloads. The components of an Integrated Manager for Lustre software file storage system include the following:

Reviewer: It looks like a simple line. Maybe worth `fmt -80` or something for easy reading and diffs.

Reviewer 2: I like the idea but it breaks several of the links and a few other things. As an alternative, I installed the markdown formatter and applied it to the document.

## Key Features

The following entries are key features of Integrated Manager for Lustre software:

## GUI-based creation and management of Lustre file systems

> The Integrated Manager for Lustre software provides a powerful, yet easy-to-use GUI that <!--enables rapid creation of Lustre file systems. The GUI supports easy configuration for high availability and expansion, and -->enables performance monitoring and management of multiple Lustre file systems. <!--See [Creating a new Lustre file system](Creating_new_lustre_fs_3_0.md/#3.0).-->In future releases, the GUI is also going to support rapid creation of Lustre file systems as well as their easy configuration for high availability and expansion.

Reviewer 3: I tried to see the difference between the original diff and the suggested one, and don't see any 🤔

> The Integrated Manager for Lustre software provides a powerful, yet easy-to-use GUI that <!--enables rapid creation of Lustre file systems. The GUI supports easy configuration for high availability and expansion, and -->enables performance monitoring and management of multiple Lustre file systems. <!--See [Creating a new Lustre file system](Creating_new_lustre_fs_3_0.md/#3.0).-->In future releases, the GUI is also going to support rapid creation of Lustre file systems as well as their easy configuration for high availability and expansion.

Reviewer 2: > In future releases, the GUI is also going to support rapid creation of Lustre file systems as well as their easy configuration for high availability and expansion.
vs
> In future releases, the GUI will support rapid Lustre file system creation, high availability configuration and support for expansion.

## Graphical charts display real-time performance metrics

Color charts display a variety of real-time performance metrics for single or multiple file systems with detailed output for both individual servers and targets. These metrics are rendered using the various [charts](#summary-of-charts).

> <!-- - [Read/Write Heat Map chart](Graphical_User_Interface_9_0.md/#9.2.1) -->

Reviewer 2: Can we delete the comments?

## Apache Hadoop\* adapter software

> Integrated Manager for Lustre software is supported by the Apache Hadoop\* adapter software, however the adapter software is a separate download. This Hadoop adapter for Lustre is compatible with the Apache Hadoop software<!--, versions 2.3 and 2.5 as of this writing-->. Hadoop software allows users who run MapReduce jobs to bypass storing data in HDFS, and store the MapReduce output directly to Lustre instead. This allows the analytical processes direct access to scientific output instead of transferring data from the compute cluster storage system to another file system. Optimizations have also been made to the shuffle step in MapReduce to take advantage of Lustre’s high-speed network access to data. Many workloads will see an overall reduction in end-to-end processing time by using the Hadoop adapter with the Integrated Manager for Lustre software file system. For more information, see [Hadoop Adaptor for Lustre](https://github.com/whamcloud/lustre-connector-for-hadoop/blob/master/Installation%20of%20HAL%20on%20Apache%20Hadoop.md).

Reviewer 2: ```suggestion
Integrated Manager for Lustre software is supported by the Apache Hadoop\* adapter software; this adapter is available for download separately. Hadoop software allows users who run MapReduce jobs to bypass storing data in HDFS and store the MapReduce output directly to Lustre instead. This allows the analytical processes direct access to scientific output instead of transferring data from the compute cluster storage system to another file system. Optimizations have also been made to the shuffle step in MapReduce to take advantage of Lustre’s high-speed network access to data. Many workloads will see an overall reduction in end-to-end processing time by using the Hadoop adapter with the Integrated Manager for Lustre software file system. For more information, see [Hadoop Adaptor for Lustre](https://github.com/whamcloud/lustre-connector-for-hadoop/blob/master/Installation%20of%20HAL%20on%20Apache%20Hadoop.md).
```

## Distributed Namespace

> Distributed Namespace (DNE) allows the Lustre metadata to be distributed across multiple metadata servers. Integrated Manager for Lustre software supports DNE1 (as of release 2.3.0.0), which supports the use of multiple MDTs. This enables the size of the Lustre namespace and metadata throughput to be scaled with the number of OSSs. This feature is going to be supported in future releases of the Integrated Manager for Lustre software GUI.

Reviewer 2: ```suggestion
Distributed Namespace (DNE) allows the Lustre metadata to be distributed across multiple metadata servers. Integrated Manager for Lustre software supports DNE1 (as of release 2.3.0.0), which supports the use of multiple MDTs. This enables the size of the Lustre namespace and metadata throughput to be scaled with the number of OSSs. This feature will be supported in future releases of the Integrated Manager for Lustre software GUI.
```

This section provides an overview of the Integrated Manager for Lustre software GUI. For a complete description of the GUI, see [Graphical User Interface](Graphical_User_Interface_9_0.md).

> The Integrated Manager for Lustre software GUI presents a set of intuitive windows that let you <!--set up, configure, monitor, and manage-->monitor and manage Lustre file systems. File system setup and configuration are going to be added in future GUI releases. The Dashboard window provides access to these capabilities. Click the following links for overview information:

Reviewer 2: ```suggestion
The Integrated Manager for Lustre software GUI presents a set of intuitive windows that let you <!--set up, configure, monitor, and manage-->monitor and manage Lustre file systems. File system setup and configuration are going to be added in future GUI releases. The Dashboard window provides access to these capabilities. Click the following links for further information:
```

- [File Usage chart](Graphical_User_Interface_9_0.md/#9.2.10)
- [Object Usage chart](Graphical_User_Interface_9_0.md/#9.2.11)
-->
- On the main Dashboard (for all file systems):
  - Filesystem Space Usage
  - I/O Performance
  - OST Balance
  - LNet Performance

- On the Server dashboard:
  - Read/Write Bandwidth
  - CPU Usage
  - Memory Usage
  - LNet Usage

- On the Filesystem dashboard:
  - Filesystem Space Usage
  - Filesystem Usage
  - OST Balance
  - MDT Usage

- On the MGT, MDT / OST dashboard:
	- Metadata Operations / I/O Performance
	- Space Usage
	- File Usage

### Management menu

The Management menu provides access to the following several windows, where you can <!--create, configure,-->monitor and manage file systems:

- **Servers** - This window lets you <!--add servers to-->view servers of the storage system <!--and configure LNet for each server,-->as well as provides server status information, and lets you start, stop, and remove servers.
<!-- - **Power Control** - This window lets you configure power control for each server. Here, you can add baseboard management controllers to configure IPMI to support server failover and also assign PDU outlets.-->

> - **File Systems** - This window lists your current file systems and provides current configuration information. <!--This window also provides access to step-by-step procedures to create and configure a file system and add system components.--> From this window, you can start, stop, or remove an entire file system, and you can start, stop, or remove management, metadata, or object storage targets.

Reviewer 2: ```suggestion
- **File Systems** - This window lists your current file systems and provides current configuration information. <!--This window also provides access to step-by-step procedures to create and configure a file system and add system components.--> From this window, a file system can be started, stopped, or removed entirely. Additionally, the same operations can be applied to management, metadata and object storage targets.
```

<!-- - **HSM** - Hierarchical Storage Management. This window displays HSM information for one or all Lustre file systems for which HSM has been configured. After configuration, the HSM Copytool chart displays a moving time-line of waiting copytool requests, current copytool operations, and the number of idle copytool workers.-->
<!-- - **Storage** - This window lets you configure and view a custom storage system appliance provided by a storage solution provider. The features on this window are specific to the appliance provided by the storage solution provider.-->
- **Users** - This window lets you configure accounts for superusers and users.

> - **Volumes** - This window provides features <!--to configure-->for primary and failover servers in file systems with servers configured for high availability. Each Lustre target corresponds to a single volume. If servers in the volume have been physically connected and then configured for high availability <!--(using this Volumes window and the Power Control window)-->, then primary and failover servers can be designated for a Lustre target. Only volumes that are not already in use as Lustre targets on local file systems are shown. A volume may be accessible on one or more servers via different device nodes, and it may be accessible via multiple device nodes on the same host.

Reviewer 2: ```suggestion
- **Volumes** - This window provides features <!--to configure-->for primary and failover servers in file systems with servers configured for high availability. Each Lustre target corresponds to a single volume. If servers in the volume have been physically connected and configured for high availability <!--(using this Volumes window and the Power Control window)-->, the primary and failover servers can be designated for a Lustre target. Only volumes that are not already in use as Lustre targets on local file systems are shown. A volume may be accessible on one or more servers via different device nodes, and it may be accessible via multiple device nodes on the same host.
```

**Note:** Job stats need to be enabled before then can be viewed. See [View Job stats](Monitoring_lustre_fs_4_0_0.md/#4.3).

![Job Stats](md_Graphics/job_stats.png) -->

### Logs

The Logs window displays log information and lets you filter events by date range, host, service, and messages from Lustre or all sources. The logs window also features <!--querying with auto-complete and--> linkable host names.

![Logs](md_Graphics/logs.png)

<!--
### Help

Help is context-sensitive; Clicking Help at the menu bar opens this Online Help to the related topic. Internet access is not required.
-->

### Activities

Activity messages provide information about the functioning and health of a managed filesystem.
<!--The Status indicator provides information about the functioning and health of each file system. Alerts are messages that indicate that the file system may be, or is, operating in a degraded mode.-->
![Activities](md_Graphics/activities.png)

> The counter next to the activity icon reflects the number of active issues with the cluster. The color of the icon will change between green, yellow, and red according to the severity of the highest active issue.

Reviewer 2: ```suggestion
The counter next to the activity icon reflects the number of active issues associated with the cluster. The color of the icon will change between green, yellow, and red according to the severity of the highest active issue.
```

Reviewer 3: I'd go for a little explanation what the `severity` is.

> There are five types of activity messages. Each has a color mark to represent severity, like grey for running commands, green for successfully executed commands, blue for info messages, yellow for warnings, and red for errors. See more information in the overview of the [Graphical User Interface](Graphical_User_Interface_9_0.md#9.6).

Reviewer 2: ```suggestion
There are five types of activity messages. Each message will be displayed with a color that represents its severity. Gray messages represent running commands, green messages represent successfully executed commands, blue messages represent general information, yellow messages represent warnings, and red messages represent errors. See more information in the overview of the [Graphical User Interface](Graphical_User_Interface_9_0.md#9.6).
```
